### ISSUE TITLE
Sonic & Allstars Racing Transformed: Crash on launch / Memory allocation failed

### ISSUE THREAD

**Issue by MadByteDE** [Joshua-Ashton/d9vk#201]
### Software information
- Settings on default (High)
- Game shows a short blackscreen and then crashes
- Game runs well with WineD3D, but performance isn't that great

### System information
- GPU: GTX 1060 6GB
- Driver: 430.26
- Wine version: Proton 4.2-9
- D9VK version: Latest Master / Version 0.13 (tested both)
- more system infos: https://gist.github.com/MadByteDE/e3977871207310f9f9acb035e8c1257c

### Log files
- d3d9.log: https://gist.github.com/MadByteDE/61c0104da87c615bedf6224a0b6c8e9b
- Steam.log: https://gist.github.com/MadByteDE/883324eafac62b57fb1e68ffc2f5604f

**Comment by Joshua-Ashton** [comment]
https://github.com/doitsujin/dxvk/issues/1100

**Comment by Joshua-Ashton** [comment]
Works fine for me here, appears to be the NV memory issue.

**Comment by MadByteDE** [comment]
Okay, I'll add my report to that issue instead. 
I just tried out some stuff like using another driver (430.26 -> 390.?? = didn't help) and changing the kernel (5.2 back to 4.19).. When I changed the kernel I manged to get in-game, but 30 seconds later it crashed and since then I'm not able to get back in. I also changed between 2 other kernels and tried the same, but I had no luck - just that one time.
I thought maybe it had something to do with the reboot of the system, but I rebooted now a couple times and wasn't able to start the game. Weird. 
Issue is closed for now.
